__tests__/app.test.js

```diff
@@ -75,3 +75,30 @@ describe("task 3. GET /api", () => {
       });
   });
 });
+
+describe("task 4. /api/articles/:article_id", () => {
+  test("GET 200 /api/articles/:article_id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
```

Comment:
You are requesting article 1, are you getting the correct data for article 1? Does this test give you the confidence that the correct data is coming back from the server?